### DISCUSSION TITLE
Local Operator Panel/Unit?

### DISCUSSION THREAD

**Discussion by RaW79** [revk/xESP32-Faikout#490] [Q&A]
Hi
some family members do not like or even are not able to operate the aircon by the Faikin web interface. So I´m brainstorming what local operator interface would be best to use?

My first idea is to minimice it to the minimum, just having some push button (on/off) and 2 push buttons to de/increase the setpoint for the automatic Faikin control. In addition could use a TM1637 (7 segment-display) to show the setpoint.
Very simple operation and the rest I´ll will do or the smart home by mqtt interacting with the Faikin.

Then I though, ok the silent-mode, the power-mode and adjusting the fan speed and/or the swing would also be sth very useful, but this will end up in to many push buttons and additional LEDs to visualize these operation modes.

So thought about using and e-ink display like the waveshare 2.9" or 4.2" which are supported by tasmota but for the actions i´ll also need several push buttons. Then I thought of using an Sonoff NS Panel for it including it´s touch screen but this is very small and not easy to operate for older people.

Creating a special operator interface might be a must, but simply reusing the current Faikin interface might also be a idea.

Nevertheless because of 24/7 operation I interested in a very low energy consumption and sth. like real touch Pads are not really an option for me.

Have you also had some ideas and solutions in mind?  Or does someone is using sth. like this?

Thx!

**Comment by TheBestMoshe** [comment]
If you are using Home Assistant already, how about using ESPHome?

You can use something like [these ESP32s with a touchscreen](https://www.aliexpress.us/item/3256806417953094.html)

You can then use the [LVGL](https://esphome.io/components/lvgl/) component to build the interface.

ESPHome makes it easy to pull in Home Assistant components, so you can easily pull in the Faikin climate entity.

I'm planning on doing something like this to combine two systems. I use Daikin ACs and hydronic heating. I want to have a single thermostat that can handle heating and cooling.

**Reply by RaW79** [reply]
Thanks. I'm not using HA but IObroker which ist also offering ESPHome Integration. To be honest, I' m in a process to eliminate my used ESPHome devices because it's for me more like a beta Integration in IOBroker and don't Like the Handling maintaining this. But will See have to find sth. 5" or 7" might be an Option for me. Regarding Energy consumption, do you know If it's possible to reduce to a Minimum? Display normally Off only active for eg 1minute by activating by Touch? So without Operation ~1-2W Energy consumption?

**Reply by RaW79** [reply]
At the moment I´m testing with a 3,5" capactive touch screen CYD (https://github.com/rzeldent/platformio-espressif32-sunton/) and using Haspmota. I really like the power consumption which is ~1,5W display on and <1W display off. The AutoOff after 30sec inactivity works great and waking up by simply giving it a touch also.
![grafik](https://github.com/user-attachments/assets/cb6fc2c6-a870-4adb-a327-bc2f7800f46c)

Developing and looking around for an effective user interface I found this example from OpenHasp:
https://www.openhasp.com/0.6.3/assets/videos/cc_sampl_climate_control.mp4

It looks very complete for selecting local control by the faikin module but also for manual or semi manual states. Useability might be a bit difficult for older or non tecnical people. What do think? Is it worth to give it a try?

**Comment by RaW79** [comment]
So skipped first tries using Haspmota because to less documentation/examples in combination with my level of skills does not work in the short term.
Switched to OpenHasp and MQTT is usable based on standard functions without huge need for individual coding.
In addition the CYD works out-of-the box much better according my personal feeling and the display configuration file to adjust anything is a nigtmare. So happy to avoid this.

So first installation is in place and working. Maybe I´ll change the front housing again by a better made one ( first try ;-) ).

![127fd64f72af2a4f89f2f50b42b9334d66594cfb_2_364x500](https://github.com/user-attachments/assets/36b3fb5e-d82e-45b6-b298-8d2b8a77c7d1)
![1dbebbc782a999a54f47a97d0d46ec4995d5b6e3](https://github.com/user-attachments/assets/a1fccee3-2833-4d41-a5aa-3c1eb63ae4b8)
![b6e49f421930118a6d89d90c8523e50bede496d5_2_394x500](https://github.com/user-attachments/assets/8c8ec674-114c-4cc3-a727-1a44b3f84644)
